src/YoutubeAtom.tsx

```diff
@@ -95,11 +373,27 @@ export const YoutubeAtom = ({
                 tabIndex={overlayImage ? -1 : 0}
             />
             {overlayImage && (
-                <YoutubeOverlay
-                    image={overlayImage}
-                    duration={duration}
-                    id={videoMeta.assetId}
-                />
+                <div className={hasUserLaunchedPlay ? hideOverlayStyling : ''}>
+                    <img
```

Comment:
Image here has no alt text, do we get passed that? The accessiblity function here is great but it a bit confusing to have a tabble image, can we not make the actual play button functional?

Reply:
Originally on frontend we would just set a background to a div for the overlay. We do not have any alt text, so I am not sure how best to proceed.

> can we not make the actual play button functional?
I assume what you mean is to turn the play button into an actual button removing the need for use to have to define tab-able buttons?